pkg/apis/core/validation/namespacedcloudprofile.go

```diff
@@ -85,3 +88,54 @@ func validateParent(parent core.CloudProfileReference, fldPath *field.Path) fiel
 
 	return allErrs
 }
+
+func validateKubernetesVersions(kubernetesSettings *core.KubernetesSettings, fldPath *field.Path) field.ErrorList {
```

Comment:
```suggestion
func validateNCPKubernetesVersions(kubernetesSettings *core.KubernetesSettings, fldPath *field.Path) field.ErrorList {
```

Or a similar naming, to not confuse it with `ValidateKubernetesVersions`.

Reply:
I will add `Nscpfl` to the name of the `NamespacedCloudProfile`-specific functions.

E.g. `validateKubernetesVersions ` → `validateNscpflKubernetesVersions`.